plugins/git/receive-app

```diff
@@ -1,18 +1,20 @@
 #!/usr/bin/env bash
 set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
 source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
+source "$PLUGIN_AVAILABLE_PATH/git/functions"
 
-git_receive_app(){
+git_receive_app() {
   declare desc="git receive-app plugin trigger"
   local trigger="git_receive_app"
   local APP="$1"; local REV="$2"
+  local cmd="git-build"
```

Comment:
I'd like to see desc/trigger/cmd at the top of the functions. Seems cleaner to me to have metadata vars at the top.